### PR TITLE
bt-65:  color change

### LINKED ISSUE #65: Change app colors according to updated colors in Design
Design link: [https://www.figma.com/file/Yd7USgP7PLvA0W794jHXWu/BSA_recruting?type=design&node-id=4120-2767&mode=design&t=WQoAICc70H5m5EtH-0](https://www.figma.com/file/Yd7USgP7PLvA0W794jHXWu/BSA_recruting?type=design&node-id=4120-2767&mode=design&t=WQoAICc70H5m5EtH-0)

### PR DESCRIPTION
**Pull request by mskmee**
One change, acording to color pallet

### CODE REVIEW

File: mobile/src/bundles/common/enums/ui/colors.enum.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const Color = {
     PRIMARY: '#274F8D',
-    SECONDARY: '#18A0FB',
+    SECONDARY: '#2E61AE',
     ERROR: '#FF2121',
     HOVER: '#0084DD',
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/enums/ui/colors.enum.ts]:
```suggestion
    HOVER: '#2E61AE',
```

File: mobile/src/bundles/common/enums/ui/colors.enum.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const Color = {
     PRIMARY: '#274F8D',
-    SECONDARY: '#18A0FB',
+    SECONDARY: '#2E61AE',
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/enums/ui/colors.enum.ts]:
```suggestion
```

Comment (mskmee) [reply mobile/src/bundles/common/enums/ui/colors.enum.ts]: i guess it's shall fail some styles, didnt check other pr's, atleast btns styles will fail, delete it and fix issues?

File: mobile/src/bundles/common/enums/ui/colors.enum.ts (hunk 1 above)
Comment (mskmee) [reply mobile/src/bundles/common/enums/ui/colors.enum.ts]: fix

File: mobile/src/bundles/common/enums/ui/colors.enum.ts (hunk 2 above)
Comment (katerynakostikova) [reply mobile/src/bundles/common/enums/ui/colors.enum.ts]: Yes

**Review by katerynakostikova — APPROVED** [review]

**Review by anogire — APPROVED** [review]

**Pull request merged**